fsd_config/form_jsons/generic/name-your-application.json

```diff
@@ -53,5 +53,6 @@
     ],
     "version": 2,
     "skipSummary": false,
-    "markAsComplete": false
+    "markAsComplete": false,
+    "name":"Digital Planning Improvement Fund"
```

Comment:
On for the future, we need to look at passing this in to make sure we can keep this generic